m-pereira/elixir-discuss
# Add AuthController to handle Ueberauth redirect back

#### What?

Add `AuthController` to handle `Ueberauth` redirect back, in action `callback`

#### Why?

This way we can have a simple authentication system

#### How to test it?

Just browse to `/auth/github`, sign in there, and check if you'll be redirected back to the application, with a success message
